lib/samson/logging.rb

```diff
@@ -23,9 +23,11 @@
     }
   end
 
-  require 'syslog/logger'
   config.logger = Syslog::Logger.new('samson')
+  config.action_cable.logger = Syslog::Logger.new('samson')
```

Comment:
reuse the logger from above ?

Reply:
ah I guess it needs a different formatter ? ... Logstash did not work ?

Reply:
yup, Logstash is not working because of call method mismatch.